Sources/Public/Animation/LottieAnimationViewInitializers.swift

```diff
@@ -78,6 +78,31 @@ extension LottieAnimationView {
     }
   }
 
+  /// Loads a Lottie animation asynchronously from the URL
+  ///
+  /// - Parameter url: The url to load the animation from.
+  /// - Parameter imageProvider: An image provider for the animation's image data.
+  /// If none is supplied Lottie will search in the main bundle for images.
+  @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
+  public convenience init(
+    url: URL,
+    imageProvider: AnimationImageProvider? = nil,
```

Comment:
I'd imagine we want to pass through the `session: URLSession = .shared` to the `LottieAnimation.loadedFrom` to allow consumers to specify custom headers etc.